packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/model/Model.java

```diff
@@ -40,11 +40,12 @@ public class Model extends TerariumAssetThatSupportsAdditionalProperties {
 	@TSOptional
 	private ModelMetadata metadata;
 
+
 	public Model() {
 		super();
 	}
 	// Copy constructor
-    public Model(Model other) {
+    public Model(final Model other) {
```

Comment:
Model.java was how I was testing this script. Its not technically related to this PR but there is now a tiny little refactor thats happened on commit :)